component/vdr/indy/go.mod

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright Scoir Inc. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+module github.com/hyperledger/aries-framework-go-ext/component/vdr/indy
+
+go 1.15
+
+require (
+	github.com/btcsuite/btcutil v1.0.1
+	github.com/flimzy/diff v0.1.7 // indirect
+	github.com/flimzy/testy v0.1.17 // indirect
+	github.com/go-kivik/couchdb v2.0.0+incompatible // indirect
```

Comment:
I wonder where these imports are originating since couchDB and mysql was moved out of the framework.

Reply:
Done.